dll/win32/browseui/basebarsite.cpp

```diff
@@ -714,28 +714,31 @@ LRESULT CBaseBarSite::OnCustomDraw(LPNMCUSTOMDRAW pnmcd)
                 REBARBANDINFO info;
                 WCHAR wszTitle[MAX_PATH];
                 DWORD index;
-                RECT rt;
+                UINT pad = GetSystemMetrics(SM_CXEDGE), leftpad = max(pad * 2, 4);
+                UINT btnw = 20, btnh = 18, btnarea = 1 + btnw + 1;
```

Comment:
Yes, that was my guess. If you have a better guess, feel free to speak up.

Reply:
I'm not a big fan of hardcoded pixel offsets. Have you tested this with 4K resolution or anything other than the usual ones?

Reply:
I'm only trying to fix the colors here. I did the system metrics thing for the text padding here because it looks better at seems to match Windows better.

Reply:
In case systems metrics were to be used, something like SM_CXBORDER or the like...